RGPosadas/LitterRally
# TASK-55: Process tfjs model output

closes #190 

This PR aims to implement model output processing so that the model can be easily used for #191 and #33 

When you call the model you get back an array of DetectionResult, which make it easy to then display these results using the HTML canvas.

To test this PR:
* login
* Go to `http://localhost:4200/camera` by clicking on the camera button
* You'll see a **temporary** test page that will allow you to run the model against an image and display some boxes around detected objects.
* Press the `Click me to detect!` button
* You should see something like this:
![image](https://user-images.githubusercontent.com/23544999/107153282-ac059a00-693a-11eb-8c3c-ff91fd9da164.png)


You can change the image that the model is running against by changing the URL in the `<img>` element. Keep in mind that, for now, the model can only detect `bottles`, `box`, `food` and `cans` correctly. Try to have an image with the same ratio as the original image I supplied, otherwise, please adjust the width and height variables so that the canvas fully covers the image
